.github/workflows/manual.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   manual-ci-format:
-    uses: ./.github/workflows/library_dafny_format.yml
+    uses: ./.github/workflows/library_format.yml
     with:
       dafny: '4.2.0'
```

Comment:
We need to pull these Dafny Version variables out to a common definition!

Reply:
Agree.